server/images.go

```diff
@@ -489,6 +499,25 @@ func CreateModel(ctx context.Context, name model.Name, modelFileDir, quantizatio
 
 				layers = append(layers, baseLayer.Layer)
 			}
+		case "controlvector":
+			// Save the gguf control vector without any conversion
+			path := strings.Trim(c.Args, " \t\r")
+
+			file, err := os.Open(realpath(modelFileDir, path))
```

Comment:
I think there is a bug here with realpath and `modelFileDir`. `modelFileDir` at least in my local testing seems to be `.` which means the relative path is being applied to where ollama serve is being ran rather than relative to the Modelfile.

Is there a more correct way to do this? I was basing this on the code for loading models and adaptors from a given path in the case above this:
```
} else if file, err := os.Open(realpath(modelFileDir, c.Args)); err == nil {
  defer file.Close()
  ...
}
```
Seems like that code would have the same issue I'm experiencing?